StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -84,6 +84,9 @@ final class PaymentSheetLoader {
                     && configuration.applePay != nil
                     && intent.isApplePayEnabled
 
+                // If Link is enabled, set the `useNewBrand`
+                LinkUI.useNewBrand = intent.elementsSession.linkSettings?.useRebrand ?? false
```

Comment:
This is a little bit confusing, because the deserlization logic defaults to true, but this defaults to false.  Ideally, we'd have one place where we have defaulting logic.

Reply:
I think the logic in my mind was like:
* If the elements/sessions response doesn't contain the bool, we return `true` (as the bool has probably been intentionally removed)
* If the elements/sessions response doesn't exist, we return `false` (because the `elements/sessions` request failed, so we don't want the new brand to leak to a user?)

But yeah, maybe it doesn't make sense. Ideally linkSettings would be non-optional, but that feels a little dangerous. I'll just simplify them and make it `true`, it's probably fine if it accidentally shows in some cases.